docs/api.md

```diff
@@ -0,0 +1,34 @@
+# ssh_scan API documentation
+
+**Last Updated:** 2016-09-19 <br>
+**Author:** agaurav77@yahoo.com <br>
```

Comment:
What's the practical value of author in this context?  We have it checked into git, so author should be self-evident with history and blame.